torch/ao/quantization/__init__.py

```diff
@@ -31,6 +32,19 @@
 from .stubs import *  # noqa: F403
 
 
+warnings.warn(
+    "torch.ao.quantization is deprecated. Plan is to: \n"
+    "1. Remove eager mode quantization (torch.ao.quantization.quantize, "
```

Comment:
I'd recommend creating an issue in torchao to track this and directly linking to the issue here.  This way we can update the messaging without touching the code.

"`torch.ao.quantization` is deprecated and will be removed in a future release.  Please see {link} for migration instructions and more details"

Reply:
done